monitoring/local_monitoring/gpu-demo/no-gpu/kruize_gpu_demo.py

```diff
@@ -0,0 +1,493 @@
+import json
```

Comment:
Please update documentation for this demo

Reply:
I'll add the ReadMe.md for this demo